### PR TITLE
Adding rhel9 in errata tests

### PR DESCRIPTION
**Pull request by akhil-jha**
Parameterizing some errata tests for rhel9.
Adding rhel9 in all the tests would require refactoring accorss api/cli/ui. 
This should atleast cover **SAT-5916** card.

### CODE REVIEW

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_errata.py

**Comment by akhil-jha** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_errata.py

File: tests/foreman/api/test_errata.py [hunk 1]
```diff
@@ -617,14 +621,15 @@ def test_positive_get_diff_for_cv_envs():
 
 @pytest.mark.skip_if_open("BZ:2013093")
```
Comment (adarshdubey-star) [inline tests/foreman/api/test_errata.py:622]: Non-blocking comments:
The BZ seems to be closed you can remove the `@pytest.mark.skip_if_open("BZ:2013093")`

**Comment by akhil-jha** [comment]
Continuing in https://github.com/SatelliteQE/robottelo/pull/9626

**Pull request closed without merging**